projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -73,11 +72,10 @@ let NEXT_ID = 0;
     providers:
         [{ provide: NG_VALUE_ACCESSOR, useExisting: IgxDatePickerComponent, multi: true }],
     // tslint:disable-next-line:component-selector
-    selector: 'igx-datePicker, igx-date-picker',
+    selector: 'igx-date-picker',
```

Comment:
I'd withhold from removing the selector fallback in patch releases after 7.1.0 TBH

Reply:
Will revert that and we won't remove the selector in this version.